TalismanSociety/signet-web
# Fix: Regressions, bugs, error and type improvements

# Description

- Fix `Threshold not a number` error for 1x Multisigs
- Race condition when updating URL params when creating a new Multisig
- Validate address input when creating/editing a Multisig, executing a txn, changing signer settings or adding a collaborator
- Revert/improve `useKnownAddresses` to only display addresses compatible to the currently selected Multisig
- Remove console.logs, improved types, 

## Type of change

Please delete options that are not relevant.


### 🧪 **How to test:**
- Paste addresses not compatible to currently selected Multisig on any address input field
- Address input field dropdown should only contain addresses that match currently selected Multisig format
- Check console for `Threshold not a number` errors



### 🖼️ **Screenshots:**

https://github.com/user-attachments/assets/f8055455-fa6e-4117-ba45-1fecacb3b553
